ui/src/Main.elm

```diff
@@ -250,7 +250,7 @@ queryUrl now query =
             Http.encodeUri (RFC3339.encode (fromTime now))
 
         parts =
-            [ "/store/query?from="
+            [ "../store/query?from="
```

Comment:
Wouldn't it be sufficient to eliminate the leading slash entirely so the request is made against relatively to the current document served: `store/query?from=`.